.github/workflows/bicep_az_deployment_group.yml

```diff
@@ -233,4 +233,5 @@ jobs:
               ESTORE_URL=${{ secrets.ESTORE_URL }} \
               PDC_INPUTTERS_EMAIL_RECIPIENT=${{ secrets.PDC_INPUTTERS_EMAIL_RECIPIENT }} \
               RATE_LIMIT_THRESHOLD=${{ vars.RATE_LIMIT_THRESHOLD}} \
-              MAX_UTILISATION_REPORT_FILE_SIZE=${{ vars.MAX_UTILISATION_REPORT_FILE_SIZE }}
+              MAX_UTILISATION_REPORT_FILE_SIZE=${{ vars.MAX_UTILISATION_REPORT_FILE_SIZE }} \
+              PORTAL_UI_URL=${{ vars.PORTAL_UI_URL }}
```

Comment:
I think using `vars` for this is a good idea, as the URL isn't secret. Even though vars are a beta feature, I think it is sound.

Note that you'll need to get @abhi-markan  to add the correct values to the different environments.

Ideally, it should be:
```
dev: https://tfs-dev-fd.azurefd.net/
feature: https://tfs-feature-fd.azurefd.net/
staging: https://tfs-staging-fd.azurefd.net/
prod: https://www.get-a-guarantee-for-export-finance.service.gov.uk/
```
(Note that I you should check the prod value.)

HOWEVER, because there isn't a `feature` environment on GitHub yet, you have a problem.
Because it isn't used on `dev` yet, you could do the following:
```
dev: https://tfs-feature-fd.azurefd.net/
staging: https://tfs-staging-fd.azurefd.net/
prod: https://www.get-a-guarantee-for-export-finance.service.gov.uk/
```

And keep track that the `dev` will need updating before the `main-stbr` branch is merged to `main`.